.gitignore

```diff
@@ -24,3 +23,20 @@ selenium
 .babelcache.json
 *.swp
 *.swo
+
+# platform dependent parts of node_modules
```

Comment:
Not settings, but those files need to be built on the platform they will run on. This is why the `npm install` command everyone runs now would be replaced with `npm rebuild`